cnf-tests/testsuites/validationsuite/cluster/validation.go

```diff
@@ -288,6 +290,127 @@ var _ = Describe("validation", func() {
 			Expect(mcpExist).To(BeTrue(), "was not able to find the xt_u32 machine config in a machine config pool")
 		})
 	})
+	Context("OpenNESS Operator for Intel FPGA PAC N3000 (Programming)", func() {
+		var n3000NodeList = &fpgav1.N3000NodeList{}
+
+		BeforeEach(func() {
```

Comment:
lets remove this you already have a part to check if the crd exists

Reply:
-  If we remove BeforeEach with skip action, any cluster without n3000 card will give 3 failed tests.
-  Would we like to place a CRD check with a skip action in the BeforeEach step?